.drone.star

```diff
@@ -37,235 +64,225 @@ config = {
-                    "pull": "always",
-                },
-            ],
-        },
-        "api-ceph": {
-            "suites": {
-                "apiAntivirus": "apiAvCeph",
-            },
-            "servers": [
-                "daily-master-qa",
-                "latest",
-            ],
-            "databases": [
-                "mariadb:10.2",
-            ],
-            "cephS3": True,
-            "extraSetup": [
-                {
-                    "name": "configure-app",
-                    "image": "owncloudci/php:7.4",
-                    "pull": "always",
-                    "commands": [
-                        "cd /var/www/owncloud/server",
-                        "wait-for-it -t 600 clamav:3310",
-                        'php occ config:app:set --value "clamav" files_antivirus av_host',
-                        'php occ config:app:set --value "daemon" files_antivirus av_mode',
-                        'php occ config:app:set --value "3310" files_antivirus av_port',
-                    ],
-                },
-            ],
-            "extraServices": [
-                {
-                    "name": "clamav",
-                    "image": "owncloudci/clamavd",
-                    "pull": "always",
-                },
-            ],
-        },
-        "core-api-acceptance": {
-            "suites": {
-                "apiAll": "core-apiAll",
-            },
-            "databases": [
-                "mysql:8.0",
-            ],
-            "servers": [
-                "daily-master-qa",
-            ],
-            "runCoreTests": True,
-            "federatedServerNeeded": True,
-            "cron": "nightly",
-            "runAllSuites": True,
-            "numberOfParts": 35,
-            "filterTags": "~@skip&&~@app-required",
-        },
-        "core-cli-acceptance": {
-            "suites": {
-                "cliAll": "core-cliAll",
-            },
-            "databases": [
-                "mysql:8.0",
-            ],
-            "servers": [
-                "daily-master-qa",
-            ],
-            "runCoreTests": True,
-            "federatedServerNeeded": True,
-            "cron": "nightly",
-            "runAllSuites": True,
-            "numberOfParts": 3,
-            "emailNeeded": True,
-            "filterTags": "~@skip&&~@app-required&&~@dbConversion",
-        },
-        "core-webui-acceptance": {
-            "suites": {
-                "webUIall": "core-webUI",
-            },
-            "databases": [
-                "mysql:8.0",
-            ],
-            "servers": [
-                "daily-master-qa",
-            ],
-            "emailNeeded": True,
-            "runCoreTests": True,
-            "federatedServerNeeded": True,
-            "cron": "nightly",
-            "runAllSuites": True,
-            "numberOfParts": 5,
-            "filterTags": "@smokeTest&&~@skip&&~@app-required",
-        },
-    },
+    # "acceptance": {
```

Comment:
what happened here?

Reply:
@grgprarup please revert the various "debug" changes.